app/views/people/_add_contact.haml

```diff
@@ -3,6 +3,7 @@
   .media
     .pull-right
       = aspect_membership_dropdown(contact, inviter, false)
-    = person_image_link(inviter, :size => :thumb_small, :class => 'img')
-    .bd
+    .media-left
+      = person_image_link(inviter, :size => :thumb_small, :class => "media-object")
```

Comment:
Just fix the hash syntax
```ruby
= person_image_link(inviter, size: :thumb_small, class: "media-object")
```
and squash the commits. Then this should be ready to merge.